parameter/criterion/include/criterion/client/CriterionInterface.h

```diff
@@ -30,17 +30,59 @@
 #pragma once
 
 #include <string>
+#include <map>
+#include <set>
 
-#include "SelectionCriterionTypeInterface.h"
+namespace core
+{
+namespace selection
+{
+namespace criterion
+{
 
-class ISelectionCriterionInterface
+class CriterionInterface
 {
 public:
-    virtual void setCriterionState(int iState) = 0;
-    virtual int getCriterionState() const = 0;
+    /** Type which represent a criterion state part */
+    using Value = std::string;
```

Comment:
Why is it defining in the client interface but never used (in the client interface) ?